exercises/twelve-days/twelve_days_test.rb

```diff
@@ -5,10 +5,9 @@ class TwelveDaysTest < Minitest::Test
   # This test is an acceptance test.
   #
   # If you find it difficult to work the problem with so much
-  # output, go ahead and add a `skip`, and write whatever
-  # unit tests will help you. Then unskip it again
-  # to make sure you got it right.
-  # There's no need to submit the tests you write, unless you
+  # output, you can temporarily add a `skip` and write whatever
+  # unit tests will help you. There's no need
+  # to submit the tests you write, unless you
```

Comment:
We can do even better, as there is only one test here.  And we really shouldn't suggest submitting the test file that is delivered to the student.  We can remove the instruction to skip the test, as it is not skipped, and is only one test.

What do you think?

Reply:
Sounds good to me. I pushed the change.

Reply:
Thanks.  I am not sure it is the best solution, but it should achieve your immediate goal (well, to some definition of "immediate", sorry about the wait!).  The generator is likely where we need to make the change so that when generated this doesn't show back up.

Now that I am paying attention to your merge requests (on this exercise anyway) it should go faster, if you are interested.